src/core/truthTable/pla_parser.cpp

```diff
@@ -0,0 +1,91 @@
+#include "core/truthTable/pla_parser.hpp"
+
+namespace syrec {
+
+    [[maybe_unused]] void pla_parser(std::istream& in, syrec::TruthTable& reader) {
+        std::string line;
+        std::regex  whitespace("\\s+");
+
+        while (in.good() && getline(in, line)) {
+            trim(line);
```

Comment:
you defined the `trim` function to return the trimmed string and do not use it here. Consider making the `trim` function return `void`.